app/frontend/Chat/MessageView/Message.svelte

```diff
@@ -33,7 +33,7 @@
       <slot name="inner-top" />
       <div class="text selectable">
         <!-- TODO Security: Jail HTML into untrusted <iframe> for additional protection. -->
-        {@html $message.html || ""}
+        <WebView title={$t`Text`} html={message.html} />
```

Comment:
```suggestion
        <WebView title={$t`Text`} html={$message.html || ""} />
```

* The `$` is important to refresh the message content. Esp. if the msg wasn't loaded before and it's loaded on-demand.
* The `|| ""` also handles msgs which have not been downloaded or are empty.